tests/conftest.py

```diff
@@ -646,20 +646,6 @@ def nodes_occupied_nics(nodes_active_nics):
     return {node: nodes_active_nics[node]["occupied"] for node in nodes_active_nics.keys()}
 
 
-@pytest.fixture(scope="session")
-def multi_nics_nodes(hosts_common_available_ports):
-    """
-    Check if nodes has any available NICs
-    """
-    return len(hosts_common_available_ports) > 1
```

Comment:
WDYT about adding an autouse with an assertion in the `network/condftest.py` about this? It will help fail faster in case the cluster does not confirm to the basic requirements.
The fixture can be named "base_requirements" to emphasize what are the checks all about. We can add here all relevant requirements that are common to the tests in this level.

Reply:
yeah, I think we should have a network sanity for network lane, that should ensure that the cluster is multi-nic. I will do that as a part of this PR.